src/operator/elemwise_op_common.h

```diff
@@ -156,11 +157,38 @@ inline bool ElemwiseStorageType(const nnvm::NodeAttrs& attrs,
     attrs, in_attrs, out_attrs);
 }
 
+template<int n_out>
+inline bool ElemwiseStorageTypeDenseOutput(const nnvm::NodeAttrs& attrs,
+                                           const Context& ctx,
+                                           std::vector<int> */*in_attrs*/,
+                                           std::vector<int> *out_attrs) {
+  CHECK_EQ(out_attrs->size(), static_cast<size_t>(n_out)) << " in operator " << attrs.name;
+  for (size_t i = 0; i < n_out; ++i) {
+    STORAGE_TYPE_ASSIGN_CHECK(*out_attrs, i, kDefaultStorage);
+  }
+  return true;
+}
+
+inline bool IdentityAttrLikeRhsStorageType(const nnvm::NodeAttrs& attrs,
```

Comment:
Looks like this is only used once. Let's not expose it in common.h

Reply:
ok